trin-beacon/src/validation.rs

```diff
@@ -262,13 +262,15 @@ impl Validator<BeaconContentKey> for BeaconValidator {
                     .read()
                     .await
                     .get_finalized_state_root()
-                    .await?;
+                    .await;
 
-                Self::state_summaries_validation(
-                    fork_versioned_historical_summaries,
-                    latest_finalized_root,
-                )
-                .await?;
+                if let Ok(latest_finalized_root) = latest_finalized_root {
```

Comment:
Is it worth logging a `warn!` here in the case that the `latest_finalized_root` is not available?

Reply:
Added a `debug!` log because this is expected to happen if the light client is not synced.